build_tools/tests/promote_from_rc_to_final_test.py

```diff
@@ -0,0 +1,330 @@
+import argparse
+import shutil
+import sys
+import os
+import glob
+from pathlib import Path
+import tempfile
+from packaging.version import Version
+from pkginfo import Wheel
+import subprocess
+import urllib
+
+sys.path.insert(0, os.fspath(Path(__file__).parent.parent.parent / "packaging"))
+import promote_from_rc_to_final
```

Comment:
Let's move to `build_tools/packaging/tests/` (we currently have tests in subfolders next to the sources, like for https://github.com/ROCm/TheRock/tree/main/build_tools/github_actions/tests)

Then this code should change to match other tests
```
sys.path.insert(0, os.fspath(Path(__file__).parent.parent))
```